types/minapp-env/tslint.json

```diff
@@ -0,0 +1,80 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "adjacent-overload-signatures": false,
+        "array-type": false,
+        "arrow-return-shorthand": false,
+        "ban-types": false,
+        "callable-types": false,
+        "comment-format": false,
+        "dt-header": false,
```

Comment:
Could you keep all these lint rules enabled? Some of them (like `dt-header`) exist to ensure publishing and updating can go smoothly.

Reply:
done it aleady.